pkgs/top-level/python-packages.nix

```diff
@@ -687,6 +687,8 @@ in {
 
   favicon = callPackage ../development/python-modules/favicon {  };
 
+  fcmaes = callPackage ../development/python-modules/fcmaes { };
```

Comment:
I think there's more f... packages later on, I think this high up in the list is for a specific class of packages. Can't remember what exactly. Search "  #" for section headers, I think.

Reply:
fixed